src/DepartmentService.Data/DepartmentRepository.cs

```diff
@@ -104,7 +104,9 @@ public async Task<(List<DbDepartment> dbDepartments, int totalCount)> FindAsync(
 
       if (!string.IsNullOrWhiteSpace(filter.NameIncludeSubstring))
       {
-        dbDepartments = dbDepartments.Where(d => d.Name.ToLower().Contains(filter.NameIncludeSubstring.ToLower()));
+        dbDepartments = dbDepartments.Where(d =>

```

Comment:
You can use IndexOf. It can be more easier

Reply:
Won't it enumerate queryable before it's needed?

Reply:
Since we have no data yet it will just modify query from ef to sql server so it won't be harder I guess

Reply:
IndexOf will be more short solution. And it is more popular solution in the internet.